source/release-notes.txt

```diff
@@ -10,6 +10,26 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.39.4
+----------------
+
+*Released September 6, 2023*
+
+Bug Fixes:
+
+- Allows ``[object Object]`` as a valid string value in TypeChecker 
+  (:issue:`COMPASS-7132`).
+- Doesn't treat non-numbers in CSV headers as array indexes 
+  (:issue:`COMPASS-7157`).
+- Limits when custom paste handling is applied; uses clipboard data when 
```

Comment:
[suggestion]

Our style guide suggests to [avoid semicolons](https://www.mongodb.com/docs/meta/style-guide/style/punctuation/#semicolons).

Maybe break this into separate sentences?